### PR TITLE
added caa accounturi

### PR DESCRIPTION
**Pull request by ounsworth**

### CODE REVIEW

**Comment by vanbroup** [comment]
Shall we just make a reference to [RFC8657](https://datatracker.ietf.org/doc/rfc8657/)?

File: draft-vanbrouwershaven-acme-auto-discovery.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ normative:
   RFC8659:
 
 informative:
+  RFC3986:
```
Comment (vanbroup) [inline draft-vanbrouwershaven-acme-auto-discovery.md:53]: Should this be deleted as it's not referenced?

**Comment by ounsworth** [comment]
> Shall we just make a reference to [RFC8657](https://datatracker.ietf.org/doc/rfc8657/)?

Basically yes, but I think there are meaningful things to say about the interactions between `accounturi` defined in RFC8657, and `priority` defined here.

File: draft-vanbrouwershaven-acme-auto-discovery.md (hunk 1 above)
Comment (ounsworth) [reply draft-vanbrouwershaven-acme-auto-discovery.md]: Done. Thanks.

File: draft-vanbrouwershaven-acme-auto-discovery.md [hunk 2]
```diff
@@ -199,7 +211,40 @@ example.com CAA 0 issue "ca1.example"
...
+
+
+~~~ dns-rr
+example.com CAA 0 issue "ca1.example; priority=1; accounturi=https://ca1.example/account/1234"
+example.com CAA 0 issue "ca1.example; priority=2; accounturi=https://ca1.example/account/1235"
+~~~
+
+Domain owners SHOULD configure their CAA records so that the `accounturi` that is intended to be used with auto-discovered ACME requests is associated with the CAA record of the lowest priority value. For example, a domain owner could configure their CAA records as follows to indicate that accounts 1234, 1235, and 1236 are all authorized to issue for `example.com`, but that account 1234 SHOULD be used for ambiguous requests.
```
[the remark this answers is not in the document]
Comment (ounsworth) [reply draft-vanbrouwershaven-acme-auto-discovery.md:230]: I'm not sure how to explain this further. 
Let's say that a customer needs to have CAA records for all three accounts -- maybe for reasons completely unrelated to ACME, they just need to be in the CAA record.

Now then, you need to use `priority` to indicate which one should be used for auto-discovered ACME requests. That is what this example is showing; that 1234 is the one to use for ambiguous incoming requests.

If you have better wording, feel free to suggest something.

[the remark this answers is not in the document]
Comment (vanbroup) [reply draft-vanbrouwershaven-acme-auto-discovery.md:230]: I will merge this for now, but I'm not sure if this particular example is helping or just causing confusion.

**Pull request merged**